src/data_race.rs

```diff
@@ -106,7 +106,7 @@ pub enum AtomicFenceOp {
 pub struct ThreadClockSet {
     /// The increasing clock representing timestamps
     /// that happen-before this thread.
-    pub clock: VClock,
+    pub(crate) clock: VClock,
```

Comment:
Should we have a common module for data_race and weak_memory so that they can share things with each other but keep it isolated from the rest of Miri?